### DISCUSSION TITLE
Unable to send service name on NewRelic via opentelemetry-collector

### DISCUSSION THREAD

**Discussion by Anshul0311** [open-telemetry/opentelemetry-rust#1974] [General]
I want to send telemetry data to NewRelic via otel-collector . I am able to see traces via trace.id but not via service.name

I am using below references 
Reference:1 For sending data to otel-collector hosted on port 4317
https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-otlp/examples/basic-otlp
I can see **service.name** in the logs getting printed under **Resource Attributes** section just like this in my otel-collector container. But this service name , I am not able to send to NewRelic

**Also, when I was doing POC with jaegar , service name was visible on Jaegar but not visible on NewRelic**

```
> `2024-05-22T20:25:42.892Z    info    TracesExporter  {"kind": "exporter", "data_type": "traces", "name": "logging", "resource spans": 2, "spans": 2}
> 2024-05-22T20:25:42.892Z    info    ResourceSpans #0
> Resource SchemaURL:
> Resource attributes:
>      -> service.name: Str(basic-otlp-example)
> ScopeSpans #0
> ScopeSpans SchemaURL:
> InstrumentationScope basic
> InstrumentationScope attributes:
>      -> scope-key: Str(scope-value)
> Span #0
>     Trace ID       : f3f6a43579f63734fe866d34d9aa0b88
>     Parent ID      : b66eacd1fcc728d3
>     ID             : af01696ea60b9229
>     Name           : Sub operation...
>     Kind           : Internal
>     Start time     : 2024-05-22 20:25:42.877134 +0000 UTC
>     End time       : 2024-05-22 20:25:42.8771425 +0000 UTC
>     Status code    : Unset
>     Status message :
> Attributes:
>      -> another.key: Str(yes)`
```

Reference:2 For sending data to NewRelic from otel-collector
https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/nr-config

Can anyone help. Do let me know incase more information is required.

**Comment by cijothomas** [comment]
This is best asked in NewRelic support channels.
